lib/data.dart

```diff
@@ -62,6 +62,55 @@ class StudentFeedback {
       this.uid, this.email, this.upvotes, this.downvotes);
 }
 
+class FeedbackComment {
+  String commentId;
+  // String feedbackId;
+  String uid;
+  String email;
+  DateTime date;
+  String text;
+
+  FeedbackComment(
+      this.commentId,
+      // this.feedbackId,
+      this.uid, this.email, this.date, this.text);
+}
+
+Stream<List<FeedbackComment>> getComments(String feedbackId) {
+  return db
+      .collection('feedback')
+      .doc(feedbackId)
+      .collection('comments')
+      .snapshots()
+      .map((snap) {
+    List<FeedbackComment> commentList = [];
+    snap.docs.forEach((doc) {
+      var commentData = doc.data();
+
+      FeedbackComment comment = new FeedbackComment(
+          doc.id,
+          // feedbackId,
+          commentData['uid'],
+          commentData['email'],
+          commentData['date'].toDate(),
+          commentData['text']
+      );
+      commentList.add(comment);
+    });
+    return commentList;
+  });
+}
+
+Future<void> submitComment(StudentFeedback f, FeedbackComment c) {
```

Comment:
Unverified users should not be able to make comments.

Reply:
well, will add it

Reply:
added